developer-experience/recipes/onboarding-guide-template.md

```diff
@@ -0,0 +1,34 @@
+# Onboarding Guide Template
+
+When developing an onboarding document for a team, it should contain details of engagement scope, team processes, codebase, coding standards, team agreements, software requirements and setup details.
```

Comment:
This is also a great document that can help people, including the customer, use for on-boarding. I find it useful to write on-boarding documents in a way that can be adapted after CSE leaves. Ideally, it need not change aside from a few roles.